src/shims/env.rs

```diff
@@ -361,3 +399,17 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         Ok(())
     }
 }
+
+// Local helper function to be used in Windows shims
+#[allow(non_snake_case)]
+fn HowWasBufferSize((success, len): (bool, u64)) -> u32 {
```

Comment:
No need to follow Windows naming schemes in our own local helper functions.
What about `windows_check_buffer_size`?

Also the doc comment should say what the function does, not (just) where it is used.